packages/firestore/test/unit/local/mutation_queue.test.ts

```diff
@@ -42,17 +47,8 @@ import * as persistenceHelpers from './persistence_test_helpers';
 import { TestMutationQueue } from './test_mutation_queue';
 import { addEqualityMatcher } from '../../util/equality_matcher';
 
-let persistence: Persistence;
-let mutationQueue: TestMutationQueue;
-
 describe('MemoryMutationQueue', () => {
-  beforeEach(() => {
-    return persistenceHelpers.testMemoryPersistence().then(p => {
-      persistence = p;
-    });
-  });
-
-  genericMutationQueueTests();
+  genericMutationQueueTests(persistenceHelpers.testMemoryPersistence);
```

Comment:
We seem to be doing this in `local_store.test.ts` as well, but this might bite us later as this loses the `this` context. I would suggest you use an array function here.

Reply:
I can change it, but `persistenceHelpers` is a module, not an instance of a class. This is a reference to a function, not a method. For this to bite us, we would need to change the type of `persistenceHelpers` to something where `this` would matter. Replacing a module with a class would I think be a significant enough change to warrant auditing the places where it was imported and catch things like this.